clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2682,19 +2698,52 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
   }
   O << "};\n\n";
 
+  O << "// array representing kernel parameters used in all kernels defined in "
+       "the\n";
+  O << "// corresponding source\n";
+  O << "static constexpr\n";
+  O << "const bool param_omit_table[] = {\n";
+  O << "  //OMIT_TABLE_BEGIN  \n";
```

Comment:
Remove the trailing spaces printed from this and the end token (OMIT_TABLE_END).